### DISCUSSION TITLE
Idea on how to flash SANT-4CH ?

### DISCUSSION THREAD

**Discussion by GreatSUN** [arendst/Tasmota#15052] [Q&A] [answered]
Hi all, I have two 4CH eWelink switches powered by 5V / USB that also support RF Switching. If somehow possible I would prefer soldering rather than throwing those into the garbage.
If someone has any idea on where to do what to get these flashed, I am open for everything.
Link zum Dev bei Amazon: [Amazon Item Seite](https://www.amazon.de/gp/product/B08RSBR6C4/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)
Platine:
![PXL_20220307_133430250](https://user-images.githubusercontent.com/5389855/157044849-6d124ed8-b2b9-4e09-8c1e-f704a9f30c47.jpg)
Falls man es nicht richtig sehen kann, der Pfeil zeigt auf den ESP8285 Chip. Das Bild ist so gemacht, dass man den Schriftzug des Chips auf dem Bild theoretisch korrekt lesen kann (von Links nach Rechts, oben ist oben, unten unten ;-))

Vielen lieben Dank für Eure Unterstützung (oder auch Verweise auf ein Forum oder so.

**Comment by Jason2866** [comment]
It seems it needs soldering on the esp pins or around.
![8-ESP8285-Pinout](https://user-images.githubusercontent.com/24528715/157062314-e33120c0-0d95-4f43-8d4f-190017dc3a04.jpg)
If you are lucky one of the buttons is connected to GPIO 0.

**Comment by GreatSUN** [comment]
First of all... Sorry, I did not use the right picture:
![PXL_20220307_133430250](https://user-images.githubusercontent.com/5389855/157093027-3c286d97-b5a5-456b-bfd2-9c851410672a.jpg)
Then... I managed to get everything powered on via the contact holes with the according pins (from left to right: GND, VCC, RXD, TXD) also connecting the two holes in the ellipse right from the antenna hole.
![PXL_20220307_181317586](https://user-images.githubusercontent.com/5389855/157094836-22c0f8b9-2143-4f41-8e67-2de0599847f1.jpg)

But it looks, whatever I do, there is no connection coming up to the Chip or I cannot get the Chip into flash mode...

I thought about connecting all the stuff directly, but from your picture, I don't really get where to connect what... Sorry, might be a little dumb :-(
I'd guess the upper ones (U0TXD/RXD) are to be connected to RX/TX. the GPIO0 to GND (to get it into flash mode).
But where to connect 3.3V and GND to power the chip?

I have the document [ESP8285 Datasheet](https://www.espressif.com/sites/default/files/documentation/0a-esp8285_datasheet_en.pdf) where I can see all the Power connectors, but there are too many to decide the right one and I am not a great guy in soldering either, so I don't want to mess all pins up ;-)

Thanks for your support!

**Comment by Jason2866** [comment]
Gnd and 3.3V can be connected (soldered) to the LDO. GND to GND and 3.3V to Output. See picture
![introduction-to-ams1117-2](https://user-images.githubusercontent.com/24528715/157101250-147263c4-7113-4488-954d-e133bbe83dbe.png)
Soldering to the ESP (RX TX) is NOT easy.

**Reply by GreatSUN** [reply]
So, it would be like this:
GND to AMS1117 GND and GPIO0
3,3V to AMS1117 Output
TX to EMS RX and RX to EMS TX (or try to use the existing RX/TX slots that might work as well).

Is that correct though?
Do you have another (better idea), rather than soldering to EMS?

**Comment by Jason2866** [comment, marked as answer]
Correct. I would check with a continuity tester if RX TX is connected to a easier reachable / solderable place.

**Reply by GreatSUN** [reply]
Thanks a lot for your support/help!
